tests/compute/test_hetero_basics.py

```diff
@@ -843,4 +843,4 @@ def test_issue_1088(index_dtype):
     # test_group_apply_edges()
     # test_local_var()
     # test_local_scope()
-    # test_issue_1088()
+    test_issue_1088('int64')
```

Comment:
Why only int64?

Reply:
This is for offline test, for CI test, both 'int32' and 'int64' is tested.